### ISSUE TITLE
Feature Request: Separate unit for opened stock items per product (+ /stockoverview quick buttons optimizations)

### ISSUE THREAD

**Issue by berrnd** [grocy/grocy#2427]
Similar to #2252 and references #2426 and #2329.

To fit every individual need, optimize the related units for the product options `Quick consume amount` and `Quick open amount` (both currently related to the product's QU stock):
- Introduce another unit `Default quantity unit opening`
- Relate the `Quick open amount` to the (new) `Default quantity unit opening`
  - Also the "quick open button" on /stockoverview
- Relate the `Quick consume amount` to the (existing) `Default quantity unit consume`
  - Also the "quick consume button" on /stockoverview

**Comment by KoenDG** [comment]
Hello, I started using this application a few days ago and currently adding a bunch of products, ran into this.

I think this issue describes what I'm looking for.

I've bought some blocks of cheese, which I then use on a cheesegrater. I know I usually cut this block into 7 pieces, over time, so I added a conversion: 1 pack -> 7 pieces.

Setting the `Default quantity unit consume` to piece became possible. But the page keeps the consume amount on pack, where I would expect it to be piece, corresponding to `Default quantity unit consume`.

Same with some chocolate bar, for which I defined a Quantity unit "Snack" and defined a conversion: 1 pack -> 8 snacks.

Same thing: able to change `Default quantity unit consume`, but `Quick consume amount` and `Quick open amount` stay on pack, which I would expect it to become Snack.

If I understand the description, this issue aims to tackle this?

**Comment by berrnd** [comment]
> If I understand the description, this issue aims to tackle this?

Of course, please also read through the referenced other threads, exactly the same example is mentioned there.

And also just again, also already mentioned there: It's already possible to consume (and purchase and do everything else) any product in any related unit, just not directly [/stockoverview](https://demo-prerelease.grocy.info/stockoverview), but on [/consume](https://demo-prerelease.grocy.info/consume) (and [/purchase](https://demo-prerelease.grocy.info/purchase) and other pages). Don't get why each and everything needs to be done on [/stockoverview](https://demo-prerelease.grocy.info/stockoverview) for some people, it's not the only page Grocy's stock management feature consists of.
